src/app/components/implementation-view/qpu-selection/qpu-selection.component.ts

```diff
@@ -734,6 +734,22 @@ export class QpuSelectionComponent implements OnInit, AfterViewInit {
       });
   }
 
+  saveResultsToQhana() : void {
+    const request = new XMLHttpRequest();
+    const form = new FormData();
+    
+    let results = JSON.stringify(this.analyzerResults);
+    form.append('results', results);
```

Comment:
you could just set the json string as the request body and in the plugin API read the body as text for the same effect.